cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="staff_only">
+                <%- gettext('Hide from students') %>
+            </label>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="hide_after_due">
+                <%- gettext('Hide content but not grades from students after due date') %>
```

Comment:
I think the most important part of this option choice is "after due date", which gets a bit lost while people are parsing "hide content but not grades". What do you think about moving the "but not grades" into the tip info and just having the option read "Hide content from learners only after due date."
(The "don't affect grades while hiding" aspect is really our behavior fix and can be explained in the tip?)